plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/RenderJob.java

```diff
@@ -344,6 +393,16 @@ private boolean isUsingLocalResources(Renderer renderer) {
             }
         }
     
+        private boolean isConflictingGeoResources(Renderer renderer){
```

Comment:
@vitalus Can you explaint the usage of RENDERING_CONFLICT_RESOLUTION_MARKER for IGeoResource.getPersistentProperties() 
Where is the property set (cannot find it in any uDig-Class)? and what is conflict resolution for?